addons/web/static/tests/views/fields/char_field_tests.js

```diff
@@ -364,17 +388,18 @@ QUnit.module("Fields", (hooks) => {
         await editInput(target, `.o_field_char input[type="text"]`, "new english value");
         await click(target, ".o_field_char .btn.o_field_translate");
 
-        enFields = target.querySelectorAll(".modal .o_translation_dialog .translation input");
+        translations = target.querySelectorAll(".modal .o_translation_dialog .translation input");
         assert.strictEqual(
-            enFields[0].value,
+            translations[0].value,
             "new english value",
             "Modified value should be used instead of translation"
         );
         assert.strictEqual(
-            enFields[enFields.length - 1].value,
+            translations[1].value,
             "valeur français",
             "French translation should be filled"
         );
+        assert.strictEqual(translations[2].value, "", "Empty spanish translation should be filled");
```

Comment:
The explanation on this line doesn't seem to agree with what's being asserted, maybe you meant "spanish translation should remain empty"?